src/Components/FormCreation/GetPreviewLinkDialog.js

```diff
@@ -0,0 +1,259 @@
+import React, { useEffect } from "react";
+import Button from "@material-ui/core/Button";
+import styled from "styled-components";
+import Dialog from "../Common/Dialogs/Dialog";
+import Close from "@material-ui/icons/Close";
+import IconButton from "@material-ui/core/IconButton";
+
+const Header = styled.div`
+  display: flex;
+  padding-bottom: 16px;
+  color: rgba(0, 0, 0, 0.87);
+  margin: 0;
+  font-weight: 500;
+  font-size: 14px;
+  line-height: 21px;
+  letter-spacing: 0.1px;
+  padding-top: 6px;
+`;
+
+const WarningMessage = styled.div`
+  display: flex;
+  padding-bottom: 16px;
+  color: rgba(0, 0, 0, 0.87);
+  background-color: #ffc27a;
+  margin: auto;
+  width: 100%;
+  padding-top: 6px;
+  height: 72px;
+  left: 2.91%;
+  right: 2.91%;
+  top: 92px;
+  border-radius: 4px;
+  text-align: left;
+`;
+
+const WarningMessageText = styled.div`
+  position: absolute;
+  left: 11.65%;
+  right: 25.12%;
+  top: 106px;
+  font-family: Roboto;
+  font-size: 14px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 21px;
+  letter-spacing: 0.25px;
```

Comment:
I also created some global CSS definitions in my PR

It has 
`
  font-family: Roboto;
  font-size: 14px;
  font-style: normal;
  font-weight: 400;
  line-height: 21px;
  letter-spacing: 0.25px;
`

Reply:
I will also handle this